pkg/controller/controller_instance.go

```diff
@@ -1166,18 +1206,6 @@ func clearServiceInstanceCurrentOperation(toUpdate *v1beta1.ServiceInstance) {
 	toUpdate.Status.OrphanMitigationInProgress = false
 	toUpdate.Status.LastOperation = nil
 	toUpdate.Status.InProgressProperties = nil
-	toUpdate.Status.ReconciledGeneration = toUpdate.Generation
-}
-
-// rollbackReconciledGenerationOnDeletion resets the ReconciledGeneration if a
```

Comment:
@jboyd01 @kibbles-n-bytes I believe we don't need this hack anymore, given that now we copy to `ObservedGeneration` from `ReconciledGeneration` instead of `instance.Generation`.

Please check.

Reply:
Yeah, that should no longer be necessary.